scripts/deployment/deploy_19_revoke_CM.js

```diff
@@ -0,0 +1,43 @@
+/*global process*/
+
+const { ethers } = require("hardhat");
+
+async function main() {
+    const fs = require("fs");
+    const globalsFile = "globals.json";
+    const dataFromJSON = fs.readFileSync(globalsFile, "utf8");
+    let parsedData = JSON.parse(dataFromJSON);
+
+    const adminRole = ethers.utils.id("TIMELOCK_ADMIN_ROLE");
```

Comment:
This line is not needed since the admin role was already revoked, and is not used later in the script

Reply:
right! Thanks